docs/CreatingIndices.md

```diff
@@ -35,6 +37,6 @@ The process to create an index is:
 
 1. `poetry install` of this library
 2. copy the `template.json` and edit it per your index
-3. `poetry run create --index-type=TYPE --config-file-path=/path/to/template/you/copied/and/edited`
+3. `MERQ_API_KEY poetry run create --index-type=TYPE --config-file-path=/path/to/template/you/copied/and/edited`
```

Comment:
MERQ_API_KEY=[env], perhaps?

Reply:
I will fix this in the coming PR today, which is greatly expaning on this readme